modules/economy-system/configs/strings.json

```diff
@@ -7,20 +7,30 @@
   "content": [
     {
       "field_name": "notFound",
-      "default": "This item could not be found",
+      "humanname-en": "not found",
+      "humanname-de": "Nicht Gefunden",
```

Comment:
I would suggest calling it something like "Not-Found-Message" / "Nicht-Gefunden-Nachricht" as it is a message